lib/fortnox/api/repositories/base/options.rb

```diff
@@ -8,6 +8,7 @@ class Options
                         :unique_id, :attr_to_json_map, :json_to_attr_map,
                         :keys_filtered_on_save
 
+          #rubocop:disable Metrics/ParameterLists
```

Comment:
Yeah ... That one is a bit of a problem. I guess I could turn it into a DSL instead of a massive constructor. But a lot of this is required anyway, or will be defined for each modell at least. I'll think about it.